### PR TITLE
build(deps): bump terraform from 1.6.6 to 1.7.5

### PR DESCRIPTION
**Pull request by HorizonNet**
https://github.com/hashicorp/terraform/releases/tag/v1.7.5
https://releases.hashicorp.com/terraform/1.7.5/

### CODE REVIEW

**Comment by jmf373** [comment]
Is this ready to be merged? I'm currently using TF 1.7.1 but would like to use Dependabot 🙌

File: terraform/Dockerfile [hunk 1]
```diff
@@ -2,13 +2,13 @@ FROM ghcr.io/dependabot/dependabot-updater-core
 ARG TARGETARCH
 
 # See https://github.com/hashicorp/terraform/releases or https://releases.hashicorp.com/terraform/
-ARG TERRAFORM_VERSION=1.6.6
+ARG TERRAFORM_VERSION=1.7.2
```
Comment (yeikel) [inline terraform/Dockerfile:5]:
```suggestion
ARG TERRAFORM_VERSION=1.7.5
```

File: terraform/Dockerfile [hunk 2]
```diff
@@ -2,13 +2,13 @@ FROM ghcr.io/dependabot/dependabot-updater-core
...
 
 # curl "https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_SHA256SUMS" | grep "terraform_${TERRAFORM_VERSION}_linux_amd64.zip"
-ARG TERRAFORM_AMD64_CHECKSUM=d117883fd98b960c5d0f012b0d4b21801e1aea985e26949c2d1ebb39af074f00
+ARG TERRAFORM_AMD64_CHECKSUM=e3965584b2347edd294ca97068af573654716abbb2ce5230d309692dab659267
```
Comment (yeikel) [inline terraform/Dockerfile:8]:
```suggestion
ARG TERRAFORM_AMD64_CHECKSUM=3ff056b5e8259003f67fd0f0ed7229499cfb0b41f3ff55cc184088589994f7a5
```

File: terraform/Dockerfile [hunk 3]
```diff
@@ -2,13 +2,13 @@ FROM ghcr.io/dependabot/dependabot-updater-core
...
 
 # curl "https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_SHA256SUMS" | grep "terraform_${TERRAFORM_VERSION}_linux_arm64.zip"
-ARG TERRAFORM_ARM64_CHECKSUM=4066567f4ba031036d9b14c1edb85399aac1cfd6bbec89cdd8c26199adb2793b
+ARG TERRAFORM_ARM64_CHECKSUM=9fe80ff9f2637d44dad78a87cc6d97f80c205b43674384ec52ba7dcd7d1e7a44
```
Comment (yeikel) [inline terraform/Dockerfile:11]:
```suggestion
ARG TERRAFORM_ARM64_CHECKSUM=08631c385667dd28f03b3a3f77cb980393af4a2fcfc2236c148a678ad9150c8c
```

**Comment by yeikel** [comment]
Let's update it to the latest (1.7.5) if possible

@abdulapopoola We need some help to move this forward

**Comment by HorizonNet** [comment]
@yeikel Yeah, a few releases happened since then. I incorporated your suggestions and updated the PR title and comment accordingly.

**Review by Nishnha — APPROVED** [review]

**Pull request merged**